### DISCUSSION TITLE
Container solectrus-influxdb-1 ist ungesund

### DISCUSSION THREAD

**Discussion by beulutz** [solectrus/solectrus#2008] [Q&A] [answered]
Hallo Herr Ledermann,
ihr tolles Projekt möchte ich auch nutzen und habe es versucht auf einem Raspberry 4 mit 8 GB RAM zu installieren. Bin auch ein ganzes Stück voran gekommen, aber nun bin ich am Ende... Habe von der Programmierung keine Ahnung. Früher habe ich mal Z80 in Assembler programmiert. Das ist aber mehr als 40 Jahre her. Nun bin ich aber immer noch ein technisch interessierter Rentner.

Nachdem alles installiert ist erhalte ich folgende Aufschrift: 

```
lutz@raspberrypi:~/solectrus $ docker komponieren 
[+] Läuft 2/0 
✔ Container solectrus-db-1 Mit 0.0s 
✔ Container solectrus-redis-1 läuft 0.0s 

Befestigung an solectrus-app-1, solectrus-db-1, solectrus-forecast-collector-1, solectrus-influxdb-1, solectrus-redis-1, solectrus-senec-collector-1 

Abhängigkeit konnte nicht gestartet werden: Container solectrus-influxdb-1 ist ungesund lutz@raspberrypi:~/solectrus $  
```

Was muss ich jetzt tun? Helfen Sie mir bitte.

Viele Grüße
Lutz

**Comment by ledermann** [comment, marked as answer]
Der etwas merkwürdig eingedeutschten Ausgabe (erscheint das wirklich so?) zufolge startet InfluxDB nicht korrekt. Ich gehe davon aus, dass Sie sich an die [Anleitung für den Raspi](https://github.com/solectrus/hosting/tree/main/guide/raspberry-pi) gehalten haben.

Um der Ursache auf die Spur zu kommen, benötige ich genauere Informationen, nützlich wäre insbesondere der Output des folgenden Befehls:

```
docker logs solectrus-influxdb-1
```

Hilfreich könnte es sein, den Ordner für die Datenablage von InfluxDB zu löschen, neu anzulegen und dann erneut zu starten. Das geht wie folgt:

```
cd /home/pi/solectrus
docker-compose down
rm -rf influxdb
mkdir influxdb
docker compose up
```

Wenn das auch nicht klappt, wäre ebenfalls der gesamte Output nützlich (einfach in die Antwort einkopieren per Copy&Paste). Je nachdem, was da zu sehen ist, kann ich dann weitere Hinweise geben.

P.S.: Mit dem Z80 hatte ich auch mal zu tun, das ist ebenso lange her :)

**Reply by beulutz** [reply]
Hallo Herr Ledermann,
habe alles nochmal neu aufgesetzt. Jetzt läuft es. Vielen Dank!
